sdk/spring/spring-cloud-azure-autoconfigure/src/test/java/com/azure/spring/cloud/autoconfigure/eventhubs/AzureEventHubsMessagingAutoConfigurationTests.java

```diff
@@ -65,10 +72,48 @@ void connectionInfoAndCheckpointStoreProvidedShouldConfigure() {
             )
             .withUserConfiguration(AzureEventHubsPropertiesTestConfiguration.class)
             .withBean(CheckpointStore.class, TestCheckpointStore::new)
+            .withBean(ObjectMapper.class)
             .run(context -> {
                 assertThat(context).hasSingleBean(EventHubsProcessorFactory.class);
                 assertThat(context).hasSingleBean(AzureEventHubsMessagingAutoConfiguration.ProcessorContainerConfiguration.class);
             });
     }
 
+    @Test
+    void withoutObjectMapperShouldNotConfigure() {
+        this.contextRunner
+            .withClassLoader(new FilteredClassLoader(ObjectMapper.class))
+            .withPropertyValues(
+                "spring.cloud.azure.eventhubs.connection-string=" + String.format(CONNECTION_STRING_FORMAT, "test-namespace")
+            )
+            .run(context -> assertThat(context).doesNotHaveBean(EventHubsMessageConverter.class));
+    }
+
+    @Test
+    void withObjectMapperShouldConfigure() {
+        this.contextRunner
+            .withPropertyValues(
+                "spring.cloud.azure.eventhubs.connection-string=" + String.format(CONNECTION_STRING_FORMAT, "test-namespace")
+            )
+            .withUserConfiguration(AzureEventHubsPropertiesTestConfiguration.class)
+            .withBean(ObjectMapper.class)
+            .run(context -> assertThat(context).hasSingleBean(EventHubsMessageConverter.class));
+    }
+
+    @Test
+    void withCustomizeObjectMapperShouldConfigure() {
+        this.contextRunner
+            .withPropertyValues(
+                "spring.cloud.azure.eventhubs.connection-string=" + String.format(CONNECTION_STRING_FORMAT, "test-namespace"),
+                "spring.jackson.serialization.write-dates-as-timestamps=true"
+            )
+            .withUserConfiguration(AzureEventHubsPropertiesTestConfiguration.class)
+            .withBean(JacksonAutoConfiguration.class)
```

Comment:
since we leverage the om bean from JacksonAutoConfiguration by default, then how about we add such auto-configuraiton as our text context default configuration?